### PR TITLE
First iteration of fetching invoice data. 

### PR DESCRIPTION
**Pull request by jsandov**
Collaborated with @laurenng , additional items will need to be reviewed down the road.

-          1/18/2022 Currently SDK query parameters don't work like expected.
-         Customer Name field is also not available.

### CODE REVIEW

File: PythonExample/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (tspence) [inline PythonExample/.idea/.gitignore:3]: Can we just ignore the .idea folder completely?  I don't think this is needed in the repository.

File: PythonExample/main.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        print('NO API KEY')
+    else:
+        print('VALID API KEY FOUND')
+        return API_KEY
+
+
+def create_client(apikey):
+    #     create client
+    env = 'https://api.sbx.lockstep.io/'
+    client = LockstepApi(env)
```
Comment (tspence) [inline PythonExample/main.py:17]: Suggestion to simplify the code: 

```
client = LockstepApi("sbx")
```

We want to encourage our customers to use the simplest possible approach and not ask them to remember to type in the complete URL each time.

File: PythonExample/main.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    while page_num < 5: # replace with True, when SDK is working, currently, it is in an endless loop otherwise
+        # Single API call to fetch invoices and company info.
+        # 1/18/2022 Currently SDK query parameters don't work like expected.
+        # Customer Name field is also not available.
+        invoices = client.invoices.query_invoices(
+            "invoiceDate GT 2021-01-10 AND invoiceDate LT 2021-03-10",
+            "Company",
+            "invoiceDate asc",
+            100,
+            20)
```
Comment (tspence) [inline PythonExample/main.py:46]: Can we please replace this with the actual page_num variable now please?

File: PythonExample/main.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    client = create_client(API_KEY)
+
+    # Ping
+    status_results = client.status.ping()
+    print(f"StatusResult: {status_results}")
+
+    page_num = 0
+    count = 1
+
+    while page_num < 5: # replace with True, when SDK is working, currently, it is in an endless loop otherwise
```
Comment (tspence) [inline PythonExample/main.py:37]: Let's not land code that includes TODO comments

File: PythonExample/main.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        # Single API call to fetch invoices and company info.
+        # 1/18/2022 Currently SDK query parameters don't work like expected.
```
Comment (tspence) [inline PythonExample/main.py:39]: Is this still broken?

File: PythonExample/main.py (hunk 4 above)
Comment (jsandov) [reply PythonExample/main.py:37]: Removed

File: PythonExample/main.py (hunk 5 above)
Comment (jsandov) [reply PythonExample/main.py:39]: We can pull the data available to us, but we are not able to grab the Company Name, only the Company Id. I believe it has to do with how our query might be interpreted. If the query is being interpreted correctly then we are missing some data that needs to be made available.

File: PythonExample/main.py (hunk 3 above)
Comment (jsandov) [reply PythonExample/main.py:46]: This is done.

File: PythonExample/main.py (hunk 2 above)
Comment (jsandov) [reply PythonExample/main.py:17]: This is done

**Comment by tspence** [comment]
LGTM!  Great!

**Pull request merged**